concepticondata/conceptlists/MorenoMartinez-2012-360.tsv

```diff
@@ -0,0 +1,362 @@
+ID	NUMBER	ENGLISH	SPANISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	SEMANTIC_CATEGORY_23	AoA_M	FAMILIARITY_LICKERT_M	TYPICALITY_LICKERT_M	VISUALCOMPLEX_LICKERT_M	LEXICALFREQ_LOG
+MorenoMartinez-2012-360-1	1	"
```

Comment:
There's a newline that you introduced in the data, probably due to the use of `"`. I think you accidentally added the newline in the word `Armadillo`.